### ISSUE TITLE
Because of unquoted path ament fails on Windows

### ISSUE THREAD

**Issue by serge-nikulin** [ament/ament_tools#160]
```
+++ Building 'ament_cppcheck'
'C:\Program' is not recognized as an internal or external command, operable program or batch file.
```
This happens because  ament_tools/ament_tools/setup_arguments.py contains the following command line construction code:

    # invoke get_setup_arguments() in a separate interpreter
    cmd = prefix + [sys.executable, '-c', '"%s"' % ';'.join(code_lines)]
    result = subprocess.run(
        ' '.join(cmd), stdout=subprocess.PIPE, shell=True, check=True)

If Python got installed into "Program Files" folder, the final  unquoted command line *cmd* can't be executed.

Proposed solution:
```
    sys_executable = sys.executable
    if not (sys_executable.startswith('"') and sys_executable.endswith('"')):
        sys_executable = '"' + sys_executable + '"'
    cmd = prefix + [sys_executable, '-c', '"%s"' % ';'.join(code_lines)]
```

**Comment by dirk-thomas** [comment]
Please create a pull request with your proposed change. That way we can simply run CI using the modified code and verify that it works.

**Comment by dirk-thomas** [comment]
Thank you for the report. The problem should be fixed by #162.

**Comment by serge-nikulin** [comment]
@dejanpan
